client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -219,7 +219,9 @@ public static URI parseServer(String server)
 
         HostAndPort host = HostAndPort.fromString(server);
         try {
-            return new URI("http", null, host.getHost(), host.getPortOrDefault(80), null, null, null);
+            int port = host.getPortOrDefault(80);
+            String schema = port == 443 ? "https" : "http";
```

Comment:
I would rename to `scheme`.

Reply:
+1, will do. I should be scheme, not schema.

Reply:
done, please see latest changes.